custom/icds_reports/ucr/data_sources/dashboard/delivery_forms.json

```diff
@@ -242,6 +242,36 @@
           "type": "property_path",
           "property_path": ["form", "next_due"]
         }
+      },
+      {
+        "column_id": "num_children_del",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "property_path",
+          "property_path": ["form", "num_children_del"]
+        }
+      },
+      {
+        "column_id": "still_birth",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "property_path",
+            "property_path": [
+              "form",
+              "child",
+              "still_live_birth"
+            ]
+          },
+          "cases": {
+            "live": 0,
+            "still": 1
+          },
+          "default": 0
```

Comment:
it may be useful to use different values for `live` and `default`. As it is right now you won't know the difference between live birth or this question was skipped. This may not be applicable for the current state of the app, but depending on how the app or this report evolves it could save some work later (and its useful to use similar patterns in other places)

I don't think we have a standard on whether to use 0/1/2 or null/0/1

Reply:
good point. null (for default) seems more logical to me assuming that's what you were proposing

Reply:
https://github.com/dimagi/commcare-hq/pull/25720/commits/0bae09c870c1c8e6232b1e2da18e55c68b177d48